dbt/models/model/model.vw_pin_shared_input.sql

```diff
@@ -79,15 +79,65 @@ housing_index AS (
     GROUP BY puma.pin10, ihs.year
 ),
 
```

Comment:
@wrridgeway 
A summary of the join:
Essentially, we want a join which accounts for the lagged variable (the census data for the most recent version is from 2021, but we want it to join with 2024 data). Then we want to allow for new data to be input, but we don't know if the data will be input in sequential years. This query identifies the largest value which is less than the year of the census year, and keeps that.